Experimental/DistributedAuthoritySample/Assets/Scripts/Physics/BaseObjectMotionHandler.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Unity.Multiplayer.Samples.SocialHub.Physics
 {
-    class BaseObjectMotionHandler : NetworkTransform, ICollisionHandler, IContactEventHandler
+    public class BaseObjectMotionHandler : NetworkTransform, ICollisionHandler, IContactEventHandler
```

Comment:
The thing is they actually need to be public so that other scripts can access them (BaseObjectMotionHandler, PhysicsObjectMotion and AvatarTransform).

Otherwise I have the following issue: Assets\Scripts\Player\AvatarTransform.cs(57,31): error CS0122: 'CameraControl.SetAvatarTransform(AvatarTransform)' is inaccessible due to its protection level

Reply:
I will try to change AvatarTransform to internal instead though maybe that works